src/include/connection.h

```diff
@@ -326,6 +326,7 @@ struct __wt_name_flag {
  */
```

Comment:
I'm happy for the timestamp set to go in this macro. But can we add a comment to this macro that is must be run within the lock? It could assert that the lock is taken too but that may be overkill.